### DISCUSSION TITLE
Changelog shows version 0.6.0. Why isn't the installer showing it?

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3518] [Q&A] [answered]
I accidentally came across the Changelog page, and saw that a version 0.6.0 (assume a stable build) is listed. But the launcher is not showing it. (I'm using the new launcher from a month or so ago.) Is it just a matter of time? So many improvements.

**Comment by tomber42** [comment]
see https://github.com/flybywiresim/a32nx/discussions/3524

**Comment by hotshotp** [comment, marked as answer]
There is no version 0.6.0, that is just a list of PRs that WILL come in 0.6.0
